src/AssertionInterface.php

```diff
@@ -14,8 +14,10 @@
     /**
      * Assertion method - must return a boolean.
      *
-     * @param  Rbac    $rbac
+     * @param  Rbac $rbac
+     * @param  string $permission
+     * @param  RoleInterface $role
```

Comment:
This PHPDoc can be removed, as everything is specified in the function signature, so these information are redundant.